app/assets/scripts/components/calculator-modal.js

```diff
@@ -149,7 +149,7 @@ const Calculator = React.createClass({
     const TopFive = data.topFiveAAL.map((building, i) => {
       return [
         <dl key={i} className='calc__list'>
-          <dt key={building['Risk Rank'] + 'dt'} className='stat__attribute stat__attribute--stocks'><span className='stat__attribute--tally'>{i + 1 + '.'}</span> {building['Description'].replace(/single|multi family/, '')}</dt>
+          <dt key={building['Risk Rank'] + 'dt'} className='stat__attribute stat__attribute--stocks'><span className='stat__attribute--tally'>{i + 1 + '.'}</span> {building['Description'].replace(/single|multi family/, '') !== 'Unknown' ? building['Description'].replace(/single|multi family/, '') : t('Unknown building type')}</dt>
```

Comment:
@nbumbarger  I haven't poked around the data to do so. If it's easy enough, that would work too.